contracts/gateway/AxelarAmplifierGateway.sol

```diff
@@ -113,15 +113,15 @@ contract AxelarAmplifierGateway is BaseAmplifierGateway, BaseWeightedMultisig, U
      * @return isLatestSigners True if provided signers are the current ones
      */
     function validateProof(bytes32 dataHash, Proof calldata proof) external view returns (bool isLatestSigners) {
-        return _validateProof(dataHash, proof);
+        isLatestSigners = _validateProof(dataHash, proof);
     }
 
     /**
      * @notice Returns the address of the gateway operator.
-     * @return The address of the operator.
+     * @return operatorAddress The address of the operator.
```

Comment:
don't think the naming is very useful here. it's implied by the function name. we should use named variables only for non-obvious cases

Reply:
this is another example of not taking the report at face value. we should only make the change that actually makes sense to us

Reply:
the report says modify only one line for `axelar-gmp-sdk-solidity`, but I reviewed all files and thought it's better to be consistent with the pattern (using named return). I agree with that `we should use named variables only for non-obvious cases`. Reviewed all files and updated.